diff --git a/chromium/third_party/perfetto/include/perfetto/ext/tracing/core/slice.h b/chromium/third_party/perfetto/include/perfetto/ext/tracing/core/slice.h
index 4ce66fdeadc..a95affe68b3 100644
--- a/chromium/third_party/perfetto/include/perfetto/ext/tracing/core/slice.h
+++ b/chromium/third_party/perfetto/include/perfetto/ext/tracing/core/slice.h
@@ -19,6 +19,7 @@
 
 #include <stddef.h>
 #include <string.h>
+#include <cstdint>
 
 #include <memory>
 #include <string>
diff --git a/chromium/third_party/perfetto/include/perfetto/tracing/tracing_backend.h b/chromium/third_party/perfetto/include/perfetto/tracing/tracing_backend.h
index 67b0d2eb51e..82711c6f362 100644
--- a/chromium/third_party/perfetto/include/perfetto/tracing/tracing_backend.h
+++ b/chromium/third_party/perfetto/include/perfetto/tracing/tracing_backend.h
@@ -19,6 +19,7 @@
 
 #include <memory>
 #include <string>
+#include <cstdint>
 
 #include "perfetto/base/export.h"
 
diff --git a/chromium/third_party/perfetto/src/trace_processor/importers/gzip/gzip_utils.h b/chromium/third_party/perfetto/src/trace_processor/importers/gzip/gzip_utils.h
index 624363ff66f..68783e879c8 100644
--- a/chromium/third_party/perfetto/src/trace_processor/importers/gzip/gzip_utils.h
+++ b/chromium/third_party/perfetto/src/trace_processor/importers/gzip/gzip_utils.h
@@ -18,6 +18,7 @@
 #define SRC_TRACE_PROCESSOR_IMPORTERS_GZIP_GZIP_UTILS_H_
 
 #include <memory>
+#include <cstdint>
 
 struct z_stream_s;
 
